content/javascript/concepts/objects/terms/is/is.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.is()'
+Description: 'This method is used to verify that two values are identical.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+  - 'Web Applications'
+Tags:
+  - 'Methods'
+  - 'Functions'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`Object.is()`** is used to verify that two values are identical. This method returns a boolean value. `true` indicates that the values are identical, and `false` indicates otherwise.
```

Comment:
```suggestion
**`.is()`** method is used to verify that two values are identical. This method returns a boolean value. `true` indicates that the values are identical, and `false` indicates otherwise.
```